### PR TITLE
Feature antd 3.0/spin indicator

### PR DESCRIPTION
**Pull request by kossel**
Added `indicator` props to `Spin` component as discussed in #7950. I'm just not sure why this has to go in antd-3.0, since there is no API modification. 😅

### CODE REVIEW

**Comment by benjycui** [comment]
CI failed.

**Comment by kossel** [comment]
I've look into the log, looks like are snapshot needs to be updated, should I?

**Comment by benjycui** [comment]
Try `npm test -- -u` to update snapshot.

File: components/table/Table.tsx [hunk 1]
```diff
@@ -509,7 +509,7 @@ export default class Table<T> extends React.Component<TableProps<T>, any> {
         changeableRowKeys.forEach(key => {
           if (selectedRowKeys.indexOf(key) < 0) {
             selectedRowKeys.push(key);
-          }else {
+          } else {
```
Comment (afc163) [inline components/table/Table.tsx]: That should not changed. It had been fixed in antd-3.0.

https://github.com/ant-design/ant-design/commit/86da4a95ce19c26fc2e7db837498cda94e7cd890

File: components/calendar/__tests__/__snapshots__/demo.test.js.snap [hunk 2]
```diff
@@ -627,7 +627,7 @@ exports[`renders ./components/calendar/demo/basic.md correctly 1`] = `
               </div>
             </td>
             <td
-              class="ant-fullcalendar-cell"
+              class="ant-fullcalendar-cell ant-fullcalendar-today ant-fullcalendar-selected-day"
```
Comment (afc163) [inline components/calendar/__tests__/__snapshots__/demo.test.js.snap]: That should not changed.

Comment (kossel) [reply components/calendar/__tests__/__snapshots__/demo.test.js.snap]: just rebased however, snapshot test fails if this is not added.

**Comment by afc163** [comment]
oops

**Comment by kossel** [comment]
额。。 git rebase antd-3.0 后 怎么一下子多了这么多东西。。。我是不是弄错什么了

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/4700#issuecomment-275305452

**Pull request closed without merging**